contributing/content-style-guide.md

```diff
@@ -438,7 +492,12 @@ Once you have determined that a screenshot will add value to the document where
 
 * Use screenshots to highlight navigation, on-screen elements, and other noteworthy product visuals.
 * Avoid using screenshots to demonstrate inputs and outputs. All code snippets and sample results should be in the documents as text fields.
+* Add images are under the `static` -> `img` folder.
 * Use concise filenames that are relevant to the content contained within. Enumerate them if they are part of a sequence.
+* Use JPEG format, which renders a better quality and lossless compression. JPEG format has a white background and is accessible in light and dark mode. 
+* Add *title=""* for all images to write a concise title of the image. For accessibility, it's important to use succinct text that is clear and complete. 
```

Comment:
Should we move [this content](https://www.notion.so/dbtlabs/Lightbox-Component-19d4496c073542868dee7bf33ff59b30) to the `contributing/` folder so we can link to it?

Reply:
good callout!